app/_src/production/init-containers.md

```diff
@@ -6,9 +6,9 @@ Due to the way that {{site.mesh_product_name}} implements transparent proxying,
 
 ### Network calls to outside of the mesh
 
-The common pitfall is the idea that it's possible to order init containers so that the mesh init container is run after other init containers.
+The common pitfall is the idea that it's possible to order init containers so that the mesh init container is running after other init containers.
```

Comment:
I think the old version is correct but I'm not a native speaker.

Reply:
Yeah, the old version is correct. It's a passive pattern.

Reply:
`is running` is easier to understand IMO

Reply:
How about just:

> The common pitfall is the idea that it's possible to order init containers so that the mesh init container runs after other init containers.

I asked the copywriter sat next to me 😆 and that was the suggestion